examples/samples/messageBrowse.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  delay,
+  SendableMessageInfo,
+  QueueClient,
+  ReceiveMode,
+  generateUuid,
+  Namespace
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const path = process.env.QUEUE_NAME || "";
+console.log("str: ", str);
+console.log("path: ", path);
+
+let ns: Namespace;
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(str);
+  const client = ns.createQueueClient(path, { receiveMode: ReceiveMode.peekLock });
+
+  // populate the queue with messages
+  await sendMessages(client);
+  // browse those messages
+  await peekMessages();
+}
+
+async function sendMessages(queueClient: QueueClient): Promise<void> {
+  const data: { lastName: string; firstName: string }[] = [
+    { lastName: "Einstein", firstName: "Albert" },
+    { lastName: "Heisenberg", firstName: "Werner" },
+    { lastName: "Curie", firstName: "Marie" },
+    { lastName: "Hawking", firstName: "Steven" },
+    { lastName: "Newton", firstName: "Isaac" },
+    { lastName: "Bohr", firstName: "Niels" },
+    { lastName: "Faraday", firstName: "Michael" },
+    { lastName: "Galilei", firstName: "Galileo" },
+    { lastName: "Kepler", firstName: "Johannes" },
+    { lastName: "Kopernikus", firstName: "Nikolaus" }
+  ];
+
+  for (let index = 0; index < data.length; index++) {
+    const element = data[index];
+    const message: SendableMessageInfo = {
+      body: `${element.firstName} ${element.lastName}`,
+      label: "Scientist",
+      timeToLive: 2 * 60 * 1000, // After 2 minutes, the queue gets cleared
```

Comment:
The TimeToLive is on the message, so after 2 minutes the message will be removed from the queue -- if there are messages in the queue with different TTL they won't be cleared. The way the current comment is written it looks like the entire queue will be cleared.

Reply:
I see -- will change the comment.